setup.py

```diff
@@ -16,8 +16,28 @@
 
 setup(name='haas',
       version='0.2rc2',
+      author='https://github.com/CCI-MOC/haas/network/members',
+      author_email = 'haas-dev-list@bu.edu',
       url='https://github.com/CCI-MOC/haas',
+      description='Hardware-as-a-Service or HaaS is a bare-metal isolation service that automates \
+allocation and management of non-virtualized compute resources',
+      long_description='HaaS allows mutually untrusting and incompatible bare metal services \
```

Comment:
Common practice is to pull the `long_description` from the README programmatically. Here's an example from one of my personal projects:

https://github.com/zenhack/git-wat/blob/master/setup.py

Feel free to lift the `_get_readme` function and related stuff from that file.